### DISCUSSION TITLE
How to automatically export media to csv file

### DISCUSSION THREAD

**Discussion by DJMic** [AzuraCast/AzuraCast#7593] [Q&A] [answered]
Is there a way to export media datas to csv file, using command line ?

**Comment by gAlleb** [comment, marked as answer]
Hi @DJMic, yes, everything is possible via command line) Also you can always look up what exactly AC does when you hit that or this button via browser console.

To export media to csv file you can send a GET via curl:

```
curl -X 'GET' \
  'https://domain/api/station/STATION_ID/files/bulk' \
  -H 'X-API-Key: a3a6ce75f70e6c7f:___YOUR_API_KEY___7c8ad48e61c6078' \
  -o output.csv 
```

You need to create an API key in your user profile and use `sttation id` in the url.

**Reply by DJMic** [reply]
Exactly what I need. Thanks a lot.
